### PR TITLE
Setup aws sysdiglabs repo

### PR DESCRIPTION
**Pull request by haresh-suresh**
Initial PR to setup github, ci & other stuff for aws repo

### CODE REVIEW

File: .github/workflows/ci-pull-request.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Get dependencies
+        run: |
+          go install github.com/onsi/ginkgo/ginkgo@latest
+      # Check if we need to build any go packages in this repo. If not, remove below.
+      # - name: Build
+      #   run: go build ./...
```
Comment (cgeers) [inline .github/workflows/ci-pull-request.yaml:55]: comment still needed or remove?

File: .github/git-chglog/config.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  title: CHANGELOG
+  repository_url: https://github.com/sysdiglabs/terraform-aws-secure
+options:
+  commits:
+  # filters:
+  #   Type:
+  #     - feat
+  #     - fix
+  #     - perf
+  #     - refactor
```
Comment (cgeers) [inline .github/git-chglog/config.yml:13]: comment still needed or remove?

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
+          draft: true
+          prerelease: false
+          body_path: RELEASE_CHANGELOG.md
+
+      # Check if we need to build using dockerfile in the release. If yes, uncomment below.
```
Comment (cgeers) [inline .github/workflows/release.yaml:47]: pretty sure this won't be needed, right?

File: CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1 @@
+* @draios/team-secure-onboarding
```
Comment (cgeers) [inline CODEOWNERS:1]: github is complaining of errors here, probably the github org is wrong.

**Review by cgeers — CHANGES_REQUESTED** [review]
left a few minor nits, but the only required change is in the codeowners file.

**Review by cgeers — APPROVED** [review]
pushed a commit through the PR. just being the change I seek. 😄 

everything else is pretty minor, so. 👍

File: .github/workflows/ci-pull-request.yaml (hunk 1 above)
Comment (haresh-suresh) [reply .github/workflows/ci-pull-request.yaml]: removed

File: .github/git-chglog/config.yml (hunk 2 above)
Comment (haresh-suresh) [reply .github/git-chglog/config.yml]: removed

File: .github/workflows/release.yaml (hunk 3 above)
Comment (haresh-suresh) [reply .github/workflows/release.yaml]: removed

**Pull request merged**